test/completion/pep0484_typing.py

```diff
@@ -536,7 +559,41 @@ class Foo(typing.TypedDict):
 #? ['setdefault']
 d.setdefaul
 
-#? 5 ["'foo'"]
+#? 5 ["'foo"]
 d['fo']
 #? 5 ['"bar"']
 d["bar"]
+
+class Bar(Foo):
+    another_variable: int
+
+    #? int()
+    another_variable
+    #? str()
+    foo
+    #? int()
+    an_int
```

Comment:
`foo` and `an_int` should be `#?`. They don't infer anything, this is not a bug, Python would pretty much say the same thing.

Reply:
Ah, I see. I've changed the check for `an_int` to `#?` and removed the check for `foo` (it's apparently a function we've defined for other tests, so it has a weird type that's tied to other tests and I think it's prudent to avoid testing for that).